doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -566,78 +566,104 @@ The query returns all the entities that match all the matching conditions (AND l
-    whose value is strictly less than that value (43).
-    This operation is only valid for attributes of type date, numbers or strings (used with
-    attributes of other types may lead to unpredictable results).
+    For an entity to match, it must contain the target property (temperature) and
+    its value must be strictly less than that value (43).
+    This operation is only valid for target properties of type date, numbers or strings (used with
+    target properties of other types may lead to unpredictable results).
 + **Greater or equal than**: `>=`. The right-hand side must be a single element, e.g. `temperature>=44`.
-    For an entity to match, it must contain an attribute with that exact name (temperature) and
-    whose value is greater than or equal to that value (44).
-    This operation is only valid for attributes of type date, numbers or strings (used with
-    attributes of other types may lead to unpredictable results).
+    For an entity to match, it must contain the target property (temperature) and
+    its value must be greater than or equal to that value (44).
+    This operation is only valid for target properties of type date, numbers or strings (used with
+    target properties of other types may lead to unpredictable results).
 + **Less or equal than**: `<=`. The right-hand side must be a single element, e.g. `temperature<=45`.
-    For an entity to match, it must contain an attribute with that exact name (temperature) and
-    whose value is less than or equal to that value (45).
-    This operation is only valid for attributes of type date, numbers or strings (used with
-    attributes of other types may lead to unpredictable results).
+    For an entity to match, it must contain the target property (temperature) and
+    its value must be less than or equal to that value (45).
+    This operation is only valid for target properties of type date, numbers or strings (used with
+    target properties of other types may lead to unpredictable results).
 + **Match pattern**: `~=`. The value matches a given pattern, expressed as a regular expression, e.g.
     `color~=ow`.
-    For an entity to match, it must contain an attribute with that exact name (color), and
-    whose value matches the string in the right-hand side, 'ow' in this example (`brown` and `yellow`
+    For an entity to match, it must contain the target property (color), and
+    its value must match the string in the right-hand side, 'ow' in this example (`brown` and `yellow`
     would match, `black` and `white` would not).
-    This operation is only valid for attributes of type string.
+    This operation is only valid for target properties of type string.
 
 The symbol `:` can be used instead of `==`.
 
-In the case of equal or unequal, if s string to match includes a `,`, you can use simple quote
+In the case of equal or unequal, if the string to match includes a `,`, you can use simple quote
```

Comment:
Actually, this line shouldn't have changes, as it is not related with the PR. Rollbacking...